scripts/daily_plot.py

```diff
@@ -70,9 +76,15 @@ def create_plot(df_plt_today, now, is_top=None):
 
     df_plt_selection_today = df_plt_today[df_plt_today.Com_Name.isin(plt_selection_today.index)]
 
+    conf = get_settings()
+
     # Set up plot axes and titles
     height = max(readings / 3, 0) + 1.06
-    f, axs = plt.subplots(1, 2, figsize=(10, height), gridspec_kw=dict(width_ratios=[3, 6]), facecolor='#77C487')
+    match conf['COLOR_SCHEME']:
+        case "dark":
+            f, axs = plt.subplots(1, 2, figsize=(10, height), gridspec_kw=dict(width_ratios=[3, 6]), facecolor='darkgrey')
```

Comment:
Could you move `f, axs = ...` out the case and only assign facecolor?

Reply:
Done.